cmd/draft/plugin_list.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/gosuri/uitable"
+	"github.com/spf13/cobra"
+
+	"github.com/Azure/draft/pkg/draft/draftpath"
+)
+
+type pluginListCmd struct {
+	home draftpath.Home
+	out  io.Writer
+}
+
+func newPluginListCmd(out io.Writer) *cobra.Command {
+	pcmd := &pluginListCmd{out: out}
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "list installed Draft plugins",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			//TODO: check on draftHome() to make sure no funky stuff with envs is happening...
```

Comment:
Is this something that still needs to be done? If so, can you clarify what is the problem at hand? :)